apps/governance/src/routes/proposals/components/vote-details/vote-buttons.tsx

```diff
@@ -35,20 +39,97 @@ interface VoteButtonsContainerProps {
 
 export const VoteButtonsContainer = (props: VoteButtonsContainerProps) => {
   const { pubKey } = useVegaWallet();
+
   const {
     appState: { decimals },
   } = useAppState();
+
   const { data, loading, error } = useVoteButtonsQuery({
     variables: { partyId: pubKey || '' },
     skip: !pubKey,
   });
 
+  const { params: networkParams } = useNetworkParams([
+    NetworkParams.governance_proposal_market_minVoterBalance,
+    NetworkParams.governance_proposal_updateMarket_minVoterBalance,
+    NetworkParams.governance_proposal_asset_minVoterBalance,
+    NetworkParams.governance_proposal_updateAsset_minVoterBalance,
+    NetworkParams.governance_proposal_updateNetParam_minVoterBalance,
+    NetworkParams.governance_proposal_freeform_minVoterBalance,
+    NetworkParams.governance_proposal_referralProgram_minVoterBalance,
+    NetworkParams.governance_proposal_VolumeDiscountProgram_minVoterBalance,
+    NetworkParams.spam_protection_voting_min_tokens,
+    NetworkParams.governance_proposal_market_requiredMajority,
+    NetworkParams.governance_proposal_updateMarket_requiredMajority,
+    NetworkParams.governance_proposal_updateMarket_requiredMajorityLP,
+    NetworkParams.governance_proposal_asset_requiredMajority,
+    NetworkParams.governance_proposal_updateAsset_requiredMajority,
+    NetworkParams.governance_proposal_updateNetParam_requiredMajority,
+    NetworkParams.governance_proposal_freeform_requiredMajority,
+    NetworkParams.governance_proposal_referralProgram_requiredMajority,
+    NetworkParams.governance_proposal_VolumeDiscountProgram_requiredMajority,
+  ]);
+
+  let minVoterBalance = null;
+
+  if (networkParams) {
+    switch (props.changeType) {
+      case 'UpdateMarket':
+      case 'UpdateMarketState':
+        minVoterBalance =
+          networkParams.governance_proposal_updateMarket_minVoterBalance;
+        break;
+      case 'NewMarket':
+        minVoterBalance =
+          networkParams.governance_proposal_market_minVoterBalance;
+        break;
+      case 'NewAsset':
+        minVoterBalance =
+          networkParams.governance_proposal_asset_minVoterBalance;
+        break;
+      case 'UpdateAsset':
+        minVoterBalance =
+          networkParams.governance_proposal_updateAsset_minVoterBalance;
+        break;
+      case 'UpdateNetworkParameter':
+        minVoterBalance =
+          networkParams.governance_proposal_updateNetParam_minVoterBalance;
+        break;
+      case 'NewFreeform':
+        minVoterBalance =
+          networkParams.governance_proposal_freeform_minVoterBalance;
+        break;
+      case 'NewTransfer':
+        // TODO: check minVoterBalance for 'NewTransfer'
+        minVoterBalance =
```

Comment:
**question (non-blocking)**: Do you still need to todo this?

Reply:
Yup! Just noticed that this is a bug that is currently on develop (I just moved the code closer to where its used), should be fixed now as correctly referencing the transfer governance proposal params